frontend/src/metabase/visualizations/echarts/cartesian/option/goal-line.ts

```diff
@@ -16,7 +16,14 @@ export function getGoalLineSeriesOption(
   const [_leftAxisKeys, rightAxisKeys] = chartModel.yAxisSplit;
 
   return {
-    type: "line",
+    type: "line", // type is irreelevant since we hide the series data with opacity 0
```

Comment:
Did not test it here, but maybe `data: []` would work here — I use it for the timeline events series

Reply:
That makes sense, I'll use that

Reply:
Done